strings/src/main/res/values-de/strings.xml

```diff
@@ -126,8 +126,11 @@
     <string name="sleep_timer.title">Schlaf gut! 🌙</string>
     <string name="sleep_timer.button.increment">Länger</string>
     <string name="sleep_timer.button.decrement">Kürzer</string>
+    <string name="auto_sleep_timer">Starte Schlummerfunktion automatisch</string>
```

Comment:
Please don't add any != en localizations yet. These will be added to weblate and then pushed automatically